openid.js

```diff
@@ -38,7 +39,7 @@ module.exports = function (RED) {
   })
 
   RED.httpAdmin.get('/openid-credentials/auth', function (req, res) {
-    if (!req.query.discovery || !req.query.clientId || !req.query.clientSecret || !req.query.id || !req.query.callback) {
+    if (!req.query.discovery || !req.query.clientId || !req.query.clientSecret || !req.query.scopes || !req.query.id || !req.query.callback) {
```

Comment:
Scope should not be mandatory:

```suggestion
   if (!req.query.discovery || !req.query.clientId || !req.query.clientSecret || !req.query.id || !req.query.callback) {
```